### PR TITLE
mods/README: Volunteer `issyl0` to assist with French post moderation

### PR DESCRIPTION
**Pull request by issyl0**
- My editor decided to trim trailing whitespace as well, apparently. 😩

### CODE REVIEW

**Review by freiheit — APPROVED** [review]

**Comment by quintessence** [comment]
Most of the PRs need to wait for the CLA before merge, but I'll merge this one (sorry for the delay, it was due to figuring out the CLA things for the Docs site).

**Pull request merged**
